pkg/test/environment.go

```diff
@@ -117,6 +117,7 @@ func NewEnvironment(ctx context.Context, env *coretest.Environment) *Environment
 	subnetProvider := subnet.NewDefaultProvider(ec2api, subnetCache, availableIPAdressCache, associatePublicIPAddressCache)
 	securityGroupProvider := securitygroup.NewDefaultProvider(ec2api, securityGroupCache)
 	versionProvider := version.NewDefaultProvider(env.KubernetesInterface, eksapi)
+	lo.Must0(versionProvider.UpdateVersion(ctx))
```

Comment:
nit: want to leave a comment here as to why we're doing this, similar to in operator?